ledger_device_sdk/src/io.rs

```diff
@@ -23,6 +23,7 @@ pub enum StatusWords {
     UserCancelled = 0x6e04,
     Unknown = 0x6d00,
     Panic = 0xe000,
+    DeviceLocked = 0x5515,
```

Comment:
Maybe this should be 0x6e05 ?

Reply:
On our JS lib, 0x5515 is what we understand as LockedDevice. Why do you think it should be 0x6e05? This is not an app error, but more like a device/OS level error. What do you think?

Reply:
Yes indeed you are right, it is not an app error. Which JS lib are you talking about ? I am ok to define an new StatusWord but why 0x5515 ? We have also SyscallError (`0x68xx`) but it is different. Is there a group of errors that could be returned as `0x55xx`?

Reply:
This is the ledger C SDK.  
![image](https://github.com/user-attachments/assets/944ab12b-38cd-46f3-84cb-4a4dfad4e7bd)

Reply:
That is the error you get if you try to interact with an app while the pin lock is enabled.

Reply:
Source:
https://github.com/LedgerHQ/ledger-secure-sdk/blob/ddfd051f46f43e48a92dc6940eebd43d77aa0e42/include/errors.h#L126

Reply:
Some more context 
![image](https://github.com/user-attachments/assets/afcb10ea-d356-40f4-a411-8e20b59d828b)

Reply:
Ok good, let's use it then !